### PR TITLE
Create database using Docker

### LINKED ISSUE #10: Create database using Docker
Create a database using MySQL and Docker.

### PR DESCRIPTION
**Pull request by zachary-mixson**
Create a database using MySQL and Docker.

### CODE REVIEW

File: database_creation.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+
```
Comment (AlexisMLopez) [inline database_creation.py]: Remove this empty line to be compliant with the Google Python Style Guide.

File: database_creation.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    except mariadb.Error as e:
+        print(f"Error connecting to MariaDB Platform: {e}")
+        sys.exit(1)
+    cur = conn.cursor()
+    show_databases(cur)
+
+def show_databases(cur):
+    cur.execute("SHOW DATABASES")
+
+    for x in cur:
```
Comment (AlexisMLopez) [inline database_creation.py]: Rename variable x to a more intent-revealing name.

**Review by AlexisMLopez — CHANGES_REQUESTED** [review]

**Review by AlexisMLopez — APPROVED** [review]

File: database-creation.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+            database="example"
+        )
+        print("connected")
+    except mariadb.Error as e:
+        print(f"Error connecting to MariaDB Platform: {e}")
+        sys.exit(1)
+    cur = conn.cursor()
+    show_databases(cur)
+
+
```
Comment (Andrew-Wilhelm) [inline database-creation.py:26]: Has to be 2 spaces per function per google python style guide

File: database-creation.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+def show_databases(cur):
+    cur.execute("SHOW DATABASES")
+
+    for database in cur:
+      print(database)
+
```
Comment (Andrew-Wilhelm) [inline database-creation.py]: Single space after the last line of code per google python style guide

File: discord-script.py [hunk 5]
```diff
@@ -1,68 +1,20 @@
...
+    print("Bot is online")
+    database_creation.setup_database()
+# example of bot command (if you write !ping in chat it will return Pong)
+@bot.command()
+async def ping(ctx): # ctx is content or context
+    await ctx.send("Pong")
 
 # runs the bot using security token
-bot.run("__DISCORD_TOKEN__")
+bot.run("MTAyOTk2NDAyODYwMjI0MTA3NQ.GYV45j.WewDCNINvI_Npyc-OzkI_MhCd5eqIC4VEfsCmw")    # replace with Discord token
```
Comment (Andrew-Wilhelm) [inline discord-script.py]: There has to be a single empty line after the last line of code per google python style guide

Comment (Andrew-Wilhelm) [inline discord-script.py]: Remove the Discord security token

File: database-creation.py [hunk 6]
```diff
@@ -0,0 +1,32 @@
+import mariadb
+import sys
+#run this command in linux to start the docker container with maria db
+    #docker run -d -p 3306:3306 -e MYSQL_DATABASE=example -e MYSQL_ROOT_PASSWORD=password -e TZ=America/Los_Angeles --name mdb103 mariadb:10.3
+
+global curr
```
Comment (Andrew-Wilhelm) [inline database-creation.py]: global variables should be declared at the module level and made internal to the module by prepending an _ to the name as per the google python style guide

File: discord-script.py [hunk 7]
```diff
@@ -1,68 +1,20 @@
...
-    print("Bot is offline")
-    await interaction.response.send_message("Bot is offline")
-    await tree.client.close()
-
+async def on_ready():
+    print("Bot is online")
+    database_creation.setup_database()
+# example of bot command (if you write !ping in chat it will return Pong)
+@bot.command()
+async def ping(ctx): # ctx is content or context
```
Comment (Andrew-Wilhelm) [inline discord-script.py]: Remove Ping Pong command

File: discord-script.py [hunk 8]
```diff
@@ -1,68 +1,20 @@
...
-
-@tree.command(name="shutdown", description="turn the bot off", guild=discord.Object(server_id))
-async def self(interaction: discord.Interaction):
-    print("Bot is offline")
-    await interaction.response.send_message("Bot is offline")
-    await tree.client.close()
-
+async def on_ready():
+    print("Bot is online")
+    database_creation.setup_database()
```
Comment (Andrew-Wilhelm) [inline discord-script.py]: Implement properly with slash commands

**Review by Andrew-Wilhelm — CHANGES_REQUESTED** [review]

File: discord-script.py (hunk 8 above)
Comment (zachary-mixson) [reply discord-script.py]: removed functionality from the file and will implement it once I have updated discord-script from man

File: database-creation.py (hunk 6 above)
Comment (zachary-mixson) [reply database-creation.py]: will be done in a later iteration this is just for the prototype

**Review by Andrew-Wilhelm — APPROVED** [review]

**Pull request merged**